fdbserver/storageserver.actor.cpp

```diff
@@ -4385,6 +4441,7 @@ ACTOR Future<Void> fetchKeys(StorageServer* data, AddingShard* shard) {
 		if (e.code() == error_code_actor_cancelled && !data->shuttingDown && shard->phase >= AddingShard::Fetching) {
 			if (shard->phase < AddingShard::Waiting) {
 				data->storage.clearRange(keys);
+				++data->counters.systemClearRange;
```

Comment:
Based on counter name convention, 
systemClearRange should be "systemClearRanges"

Reply:
Done.